README.md

```diff
@@ -1,85 +1,341 @@
+
+		Swift:
+		
+		```swift
+    	func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
+        	guard let url = userActivity.webpageURL else {
+        		return false
+        	}
+
+        	// This will track the click, retrieve the original URL and call `handleIterableURL:context:` with the original URL
+        	return IterableAPI.handle(universalLink: url)
+    	}
+		
+		```
+
+		Objective-C:
+		
+		```objective-c
+    	- (BOOL)application:(UIApplication *)application continueUserActivity(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray *restorableObjects))restorationHandler {
+    		// This will track the click, retrieve the original URL and call `handleIterableURL:context:` with the original URL
+		    return [IterableAPI handleUniversalLink:userActivity.webpageURL];
+		}
+		```
+		
+		Alternatively, call `getAndTrackDeeplink` along with a callback to handle the original deeplink url. You can use this method for any incoming URLs, as it will execute the callback without changing the URL for non-Iterable URLs.
+
+		Swift:
+		
+		```swift
+		func application(_ application: UIApplication, continue userActivity: NSUserActivity,
+                  restorationHandler: @escaping ([Any]?) -> Void) -> Bool {
+        	guard let url = userActivity.webpageURL else {
+        		return false
+        	}
+        	
+        	IterableAPI.getAndTrack(deeplink: url) { (originalUrl) in
+        		// Handle original url deeplink here
+        	}
+        	return true
+		}
+		```
+
+		Objective-C:
+		
+		```objective-c
+		- (BOOL)application:(UIApplication *)application
+		 		continueUserActivity(NSUserActivity *)userActivity 
+		 		restorationHandler:(void (^)(NSArray *restorableObjects))restorationHandler {
+		    
+		    [IterableAPI getAndTrackDeeplink:iterableLink callbackBlock:^(NSString* originalURL) {
+		        //Handle Original URL deeplink here
+		    }];
+		    
+		    return true;
+		}
+		```
+		
+3. ##### InApp Notifications
+	To display the user's InApp notifications call `spawnInAppNotification` with a defined `ITEActionBlock` callback handler. When a user clicks a button on the notification, the defined handler is called and passed the action name defined in the InApp template.
+	
+	InApp opens and button clicks are automatically tracked when the notification is called via `spawnInAppNotification`. Using `spawnInAppNotification`, the notification is consumed and removed from the user's in-app messages queue. If you want to retain the messages on the queue, look at using `getInAppMessages` directly. If you use `getInAppMessages` you will need to manage the in-app opens manually in the callback handler.		
+4. ##### Tracking Custom Events
+	Custom events can be tracked using `IterableAPI.track(event:...)` calls.
+	
+5. ##### Updating User Fields
+	User fields can be modified using `IterableAPI.updateXYZ(...)` calls.
```

Comment:
Isn't it just `updateUser`?

Reply:
You are right. I will make those changes to the API. I think it is OK to make it with this PR.